.circleci/continue_config.yml

```diff
@@ -55,7 +55,7 @@ commands:
           name: Install kedro and test requirements
           command: |
             cd <<parameters.plugin>>
-            pip install git+https://github.com/kedro-org/kedro@main
+            pip install git+https://github.com/kedro-org/kedro@develop
```

Comment:
You can undo these already cause `develop` has hit `main`, but it's no rush - they'll be in sync for a good while.